web/.rsync-filter

```diff
@@ -0,0 +1,6 @@
+# Retain this file to protect files from being deleted by rsync. This file
+# should be placed in the Drupal root directory and specify relative paths from
+# the Drupal root.
+- sites/default/files
+- *.scss
+- *.es6.js
```

Comment:
This seems a bit aggressive, never know what is going to be included in core/contrib. Might want to make these into `modules/custom/**/*.es6.js` (not sure if globs work here, though)